db/migrate/20230713050201_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.text :Text
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
```

Comment:
- [ ] Kindly follow Ruby convention for data attributes, `Title` should be `title`, `Text` should be `text`, `CommentsCouter` should be `comments_counter`, `LikesCounter` should be `likes_counter`

Reply:
Attributes updated